docs/core/compatibility/containers/8.0/aspnet-port.md

```diff
@@ -85,6 +87,8 @@ There are two ways to respond to this breaking change:
 - Recommended: Explicitly set the `ASPNETCORE_HTTP_PORTS`, `ASPNETCORE_HTTPS_PORTS`, and `ASPNETCORE_URLS` environment variables to the desired port. Example: `docker run --rm -it -p 9999:80 -e ASPNETCORE_HTTP_PORTS=80 <my-app>`
 - Update existing commands and configuration that rely on the expected default port of port 80 to reference port 8080 instead. Example: `docker run --rm -it -p 9999:8080 <my-app>`
 
+Apps build using the older [`WebHost.CreateDefaultBuilder`](https://learn.microsoft.com/dotnet/api/microsoft.aspnetcore.webhost.createdefaultbuilder) will need to either set `ASPNETCORE_URLS` (not `ASPNETCORE_HTTP_PORTS`) or update the expected default port of port 80 to reference port 5000 instead.
```

Comment:
I believe this is incorrect. Port 5000 is loopback so a port mapping of `8000:5000` won't work as expected. Right?

Reply:
Right. This matches my comment: https://github.com/dotnet/docs/pull/37661#discussion_r1367603520

Users must either call `.UseUrls("http://*:8080")` or set `ASPNETCORE_URLS=http://*:8080` to set the port to expose.

Reply:
I've proposed new text.